CONTRIBUTING.rst

```diff
@@ -13,21 +10,11 @@ a demonstrable problem that is caused by the REANA code, please:
    latest `master` branch.
 3. Create an issue, ideally with **a test case**.
 
-Pull requests
--------------
-
-If you create a feature branch, you can run the tests to ensure that everything
-is operating correctly:
+If you create a pull request fixing a bug or implementing a feature, you can, you can run
```

Comment:
🤦‍♂️ Sorry I've made a mess with the recipes, we need to change this everywhere...

```suggestion
If you create a pull request fixing a bug or implementing a feature, you can run
```